interface/forms/group_attendance/functions.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * interface/forms/group_attendance/functions.php functions for form
+ *
+ * Contains the functions used for the group attendance form
+ *
+ * Copyright (C) 2016 Shachar <shachar058@gmail.com>
+ * Copyright (C) 2016 Amiel<amielboim@gmail.com>
```

Comment:
Copyright is important to make clear. Add your last names (like in the below author tags) and ensure there is a space between the name and the email tag.